adapters/triplelift_native/triplelift_native_test.go

```diff
@@ -0,0 +1,10 @@
+package triplelift_native
+
+import (
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"testing"
+)
+
+func TestJsonSamples(t *testing.T) {
+	adapterstest.RunJSONBidderTest(t, "triplelifttest", NewTripleliftNativeBidder(nil, "http://tlx.3lift.net/s2s/auction?supplier_id=19", "{\"publisher_whitelist\":[]}"))
```

Comment:
Shouldn't it be `triplelift_nativetest` like the following?
```
adapterstest.RunJSONBidderTest(t, "triplelift_nativetest", NewTripleliftNativeBidder(nil, "http://tlx.3lift.net/s2s/auction?supplier_id=19", "{\"publisher_whitelist\":[]}"))
```

Reply:
yep